tests/test_auth.py

```diff
@@ -284,9 +289,16 @@ def test_jwt_decode_params_verify_false(self, request_mock, jws_mock):
             'access_token': 'access_token'
         }
         request_mock.post.return_value = post_json_mock
-        calls = [
-            call('token', 'client_secret')
-        ]
+        # Whichever Python version you use, always expect the jws_mock
+        # to be called with byte strings.
```

Comment:
This comment should perhaps be replicated around line 257 in this same file. Or is it too obvious to include at all?